tests/topotests/lib/common_config.py

```diff
@@ -31,14 +31,16 @@
 from tempfile import mkdtemp
 
 import os
-import io
 import sys
 import traceback
 import socket
 import ipaddress
+import ipaddr
```

Comment:
ipaddr was superseeded by the ipaddress module. With the work on getting topotests to run on Python3 (with limited ipaddr support), is there a specific need to use ipaddr instead of ipaddress?
If not, then please use ipaddress instead

Reply:
I have replaced ipaddr with ipaddress. Thanks @mwinter-osr